### PR TITLE
📝Docs - Guide to migrate from WordPress

### LINKED ISSUE #2386: 📝 Guide: How to migrate from Wordpress to TinaCMS
Write a guide on migrating from Wordpress to TinaCMS
https://tina.io/docs/guides/

### PR DESCRIPTION
**Pull request by Lewkans**
Closes #2386 
### General Contributing:

- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (bradystroud) [inline content/docs/guides/converting-wordpress-to-tina.mdx:57]: Can we have a MacOS command as well?

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (isaaclombardssw) [inline content/docs/guides/converting-wordpress-to-tina.mdx:70]: This is a big reduction – what kind of data do we get in these WP markdown files specifically? Should they set up block based editing? How can we look at the converted data and turn it into a schema?

Do we want a seperate guide that goes over general migration tips (ex. this is the markdown we have, how can we create a schema that fits it) – then we could just link that and use the wordpress markdown as the example

**Counterargument**
This guide looks really straightforward as is – which is great.

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (isaaclombardssw) [inline content/docs/guides/converting-wordpress-to-tina.mdx:126]: Should probably be included in the guide

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (isaaclombardssw) [inline content/docs/guides/converting-wordpress-to-tina.mdx:127]: Should probably be included in the guide

**Review by isaaclombardssw — CHANGES_REQUESTED** [review]
Really good v1

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:14]: The content returned from the rendered page is in html and css

image

Comment (isaaclombardssw) [reply content/docs/guides/converting-wordpress-to-tina.mdx:14]: Oh wow

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:57]: Done - Added mac/linux instructions

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - Updated title

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - changed the callout to link to the discord server

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - Added a second paragraph in the introduction to mention that the guide transfers content and media and doesn't include page styling

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - Added to the guide

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - Added to the guide

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - Added note box explaining this

**Review by landonmaxwell — APPROVED** [review]
LGTM - once approved by Wicksy it's good to go 👍

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: As per my conversation with @Marxoz, I've removed the 💡Note part, but kept the content as is.

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:82]: As per my conversation with @Marxoz, I've created a new PBI to create a general migration guide, see #2438.

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Ben0189) [inline content/docs/guides/converting-wordpress-to-tina.mdx:22]: i think basic understanding of Tina is also a prerequisties, maybe a link to where to learn Tina

**Review by Ben0189 — CHANGES_REQUESTED** [review]

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Ben0189) [inline content/docs/guides/converting-wordpress-to-tina.mdx:115]: is the starter typescript of js ?

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:22]: Done - added prerequisite to have a basic understanding of the editor and the schema, with links to them

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (wicksipedia) [inline content/docs/guides/converting-wordpress-to-tina.mdx:69]: why does the macOS/nix command deal with `txt` files but the windows one deal with `md` files? Should they be the same?

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (wicksipedia) [inline content/docs/guides/converting-wordpress-to-tina.mdx:75]:
```suggestion
> Note: This works because the format of WordPress posts matches the format that the Tina Starter uses.
```

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Ben0189) [inline content/docs/guides/converting-wordpress-to-tina.mdx:19]: from your below example, looks like you also need some react basic, you can link to there

**Review by Ben0189 — CHANGES_REQUESTED** [review]

**Review by wicksipedia — APPROVED** [review]
test pass with changes

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - You are absolutely right, I forgot to change it from my testing, fixed it to convert from .txt to .md to .md to .mdx

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:75]: As per my conversation with @Marxoz, there shouldn't be "Note: " at the start to remain consistent with how the other pages on the docs are set out

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx]: Done - changed from js to ts code block

File: content/docs/guides/converting-wordpress-to-tina.mdx
Comment (Lewkans) [reply content/docs/guides/converting-wordpress-to-tina.mdx:19]: Done - added prerequisite to have a basic understanding of react

**Review by isaaclombardssw — APPROVED** [review]
I'm not reading it again because you have 50,000,000 approvals :)

**Pull request merged**
